samples/bluetooth/hci_usb/README.rst

```diff
@@ -21,3 +21,11 @@ This sample can be found under :zephyr_file:`samples/bluetooth/hci_usb` in the
 Zephyr tree.
 
 See :ref:`bluetooth samples section <bluetooth-samples>` for details.
+
+.. note::
+   If you want to run this sample on the nRF5340 device (build using
+   ``-DBOARD=nrf5340dk_nrf5340_cpuapp`` or
+   ``-DBOARD=nrf5340dk_nrf5340_cpuappns``) you must additionally build
+   and program the corresponding sample for the nRF5340 network core:
+   :ref:` bluetooth-hci-rpmsg-sample` which implements the bluetooth
+   low energy controller.
```

Comment:
```suggestion
   ``-DBOARD=nrf5340dk_nrf5340_cpuappns``) you must also build
   and program the corresponding sample for the nRF5340 network core
   :ref:` bluetooth-hci-rpmsg-sample` which implements the Bluetooth
   Low Energy controller.
```

Reply:
Thanks, fixed.

Reply:
One more extra colon after network core. I believe we don't need it.